index.js

```diff
@@ -0,0 +1,56 @@
+var express = require("express");
+var hbs = require("express-handlebars");
+
+var compliments = [
+  "Your instructors love you",
+  "High five = ^5",
+  "Is it Ruby Tuesday yet?",
+  "It's almost beer o'clock",
+  "The Force is strong with you",
+  "You're a monster",
+  "Ain't found a way to kill me yet",
+  "U R 2 KUTE",
+  "Holy diver, you've been down too long in the midnight sea"
+];
```

Comment:
Data tends to be kept in a separate file. This works great, but try refactoring it so this is stored in a separate JSON file. From there, all you need to do is `require` it in here.